Jenkinsfile

```diff
@@ -27,59 +27,21 @@ pipeline {
     // global timeout
     timeout(time: 20, unit: 'MINUTES')
   }
-  agent {
-    kubernetes {
-      label "${env.label}"
-      defaultContainer 'jnlp'
-      yaml """
-apiVersion: v1
-kind: Pod
-metadata:
-spec:
-  volumes:
-    - name: maven-repo-shared-storage
-      persistentVolumeClaim:
-       claimName: glassfish-maven-repo-storage
-    - name: maven-repo-local-storage
-      emptyDir: {}
-  containers:
-  - name: jnlp
-    image: jenkins/jnlp-slave:alpine
-    imagePullPolicy: IfNotPresent
-    env:
-      - name: JAVA_TOOL_OPTIONS
-        value: -Xmx1G
-    resources:
-      limits:
-        memory: "1Gi"
-        cpu: "1"
-  - name: build-container
-    image: ee4jglassfish/ci:jdk-8.181
-    args:
-    - cat
-    tty: true
-    imagePullPolicy: Always
-    volumeMounts:
-      - mountPath: "/home/jenkins/.m2/repository"
-        name: maven-repo-shared-storage
-      - mountPath: "/home/jenkins/.m2/repository/org/glassfish/hk2"
-        name: maven-repo-local-storage
-    resources:
-      limits:
-        memory: "7Gi"
-        cpu: "3"
-"""
-    }
-  }
   stages {
     stage('build') {
+      agent any
+      tools {
+        jdk 'temurin-jdk8-latest'
+        maven 'apache-maven-latest'
+      }
       steps {
-        container('build-container') {
-          timeout(time: 10, unit: 'MINUTES') {
-            sh 'mvn clean install'
-            junit testResults: '**/target/surefire-reports/*.xml', allowEmptyResults: true
-          }
-        }
+        sh 'mvn clean install -P jacoco'
+        junit testResults: '**/target/surefire-reports/*.xml', allowEmptyResults: true
```

Comment:
Not sure if it is used in hk2, probably not, I usually use *-reports, as failsafe plugin uses it's own directory. If failsafe is not used here, just ignore my comment ;)